pinot-core/src/main/java/com/linkedin/pinot/core/operator/filter/CompositeOperator.java

```diff
@@ -15,13 +15,11 @@
  */
 package com.linkedin.pinot.core.operator.filter;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import com.linkedin.pinot.core.common.BlockId;
 import com.linkedin.pinot.core.common.Operator;
 import com.linkedin.pinot.core.operator.blocks.BaseFilterBlock;
 import com.linkedin.pinot.core.operator.blocks.CompositeBaseFilterBlock;
+import java.util.ArrayList;
+import java.util.List;
 
 public class CompositeOperator extends BaseFilterOperator{
```

Comment:
Space ? `BaseFilterOperator {`

Reply:
This class will be removed in the star-tree PR, but good catch